src/core/redux/features/album/ablum.slice.ts

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from '@reduxjs/toolkit';
+import type { PayloadAction } from '@reduxjs/toolkit';
+
+import type { RootState } from '@/core/redux/store.redux';
+import { IAlbum } from '@/core/common/interfaces/index.interface';
+import { EScopeSongStore, ESelectReducer } from '@/core/common/constants/index.constant';
+
+const initialState: IAlbum = {
+    _id: '',
+    composerReference: '',
+    title: '',
+
+
+};
+
+export const albumSlice = createSlice({
+    name: ESelectReducer.SONG,
+    initialState,
+    reducers: {
+        pushListSuggestSongIntoStore: (state, action: PayloadAction<IAlbum[]>) => {
+            state..suggests = action.payload;
```

Comment:
1. syntax error state.. 
2.  don't set initialState like that, it will have different keys and values